src/main/protobuf/v1/purpose.proto

```diff
@@ -21,6 +21,8 @@ message PurposeV1 {
   optional RiskAnalysisFormV1 riskAnalysisForm = 10;
   required int64 createdAt = 11;
   optional int64 updatedAt = 12;
+  required bool isFreeOfCharge = 13;
```

Comment:
This should be optional for retro-compatibility with existing events.
The constraint can be enforced during protobuf-persistence mapping (in `protobufUtils`)
```suggestion
  optional bool isFreeOfCharge = 13;
```

Reply:
The default value for existing events must be `true`